installers/helm/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
this file was generated by `helm create` when the chart was created. Is this something that we should keep?

Reply:
No